.eslintrc.json

```diff
@@ -0,0 +1,20 @@
+{
+    "extends": [
+        "eslint:recommended",
+        "prettier"
+    ],
+    "env": {
+        "es6": true,
+        "browser": true,
+        "webextensions": true
+    },
+    "parser": "babel-eslint",
+    "rules": {
+        "no-unused-vars": ["error", {
+            "args": "after-used",
+            "argsIgnorePattern": "^_"
```

Comment:
This makes it so that you can prefix arguments with `_` to mark them as unused and silence warnings, and in doing so makes the code more readable.

Example:
```js
function (arg1, _arg2, arg3) {
    useArgs(arg1, arg3);
}
```